valohai_yaml/objs/pipelines/execution_node.py

```diff
@@ -1,30 +1,51 @@
+from enum import Enum
 from typing import Any, Dict, List, Optional, Union
 
 from valohai_yaml.lint import LintResult
 from valohai_yaml.objs.pipelines.node import ErrorAction, Node
 from valohai_yaml.objs.pipelines.node_action import NodeAction
-from valohai_yaml.types import LintContext, NodeOverrideDict
+from valohai_yaml.types import LintContext, NodeOverrideDict, SerializedDict
+
+
+class Mode(Enum):
```

Comment:
This is a too generic name – `OverrideMode`?

I'd probably also put it in a module of its own (`valohai_yaml.objs.pipelines.override_mode`?)

Reply:
Agree about name, OverrideMode sounds much better. When it comes to location, we have ErrorActions inside node.py so I did same, but I am not sure. Maybe all localish field should be in file  eg util_classes or smth like it?